common/traits/wc_race_traits.txt

```diff
@@ -718,6 +718,60 @@ creature_lich = {
 	random = no
 	customizer = no
 }
+creature_wickerman = {	# Throsic ghoul counterpart
```

Comment:
Also, the race traits aren't added in the scripted triggers. `has_prev_race_trigger`, `require_cannot_marry_flag_trigger`, etc. 
P.S. Search for `creature_abomination` to find all the instances where the wicker constructions may need adding.

Reply:
Scripted triggers are done (SHA: 33851838171ae7aaf7e6b99fb0cf06ee1f248f35)